Scripts/Python/ki/__init__.py

```diff
@@ -4936,10 +4937,11 @@ def BigKIDisplayMarkerGame(self):
                     hSpans = coord.getHSpans()
                     vSpans = coord.getVSpans()
                     mlbMarkerList.addStringW("[{}:{},{},{}] {}".format(FilterAgeName(age), torans, hSpans, vSpans, xCensor.xCensor(desc, self.censorLevel)))
+                    showScroll = True
                 mlbMarkerList.unlock()
 
                 # Refresh the scroll position
-                self.BigKIMarkerListScrollVis(True)
+                self.BigKIMarkerListScrollVis(showScroll)
```

Comment:
This could be simplified with something like:

```suggestion
                self.BigKIMarkerListScrollVis(bool(mgr.marker_total))
```